netcore/System.Private.CoreLib/src/System/TypeNameParser.cs

```diff
@@ -125,7 +125,8 @@ static Type ResolveType (Assembly assembly, List<string> names, Func<Assembly, s
 		{
             Type type = null;
 
-            string name = EscapeTypeName (names [0]);
+            //string name = EscapeTypeName (names [0]);
```

Comment:
This doesn't seem right.

In CoreCLR, they still have a call to `EscapeTypeName` here.  I think perhaps on Mono the caller of this function is passing some pre-escaped input.

https://github.com/dotnet/coreclr/blob/4aa65f077778c7c05211825f98684aae9c52caaf/src/System.Private.CoreLib/src/System/TypeNameParser.cs#L228